tests/from_numpy.rs

```diff
@@ -11,201 +11,159 @@ mod assert;
 
 /// Test conversion of a numpy array to a Matrix3<f64>.
 #[test]
-#[rustfmt::skip]
```

Comment:
I added these on purpose to prevent the matrices from being squashed to a single line. I'm guessing this is the result of automatic merge conflict resolving, but could you restore the formatting of the matrices in the tests?

Reply:
Whoops, deficiently merge resolving. Reverted and restored matricies.